src/lib.rs

```diff
@@ -573,5 +573,41 @@ pub mod pallet {
 			Votes::<T>::remove(supersig_index, call_index);
 			UsersVotes::<T>::remove_prefix((supersig_index, call_index), None);
 		}
+
+		pub fn add_to_supersig(supersig_idx: u128, members: Vec<(T::AccountId, Roles)>) -> u128 {
+			let mut number_added = 0;
+			for (member, role) in members {
+				if Self::members(supersig_idx, &member) == Roles::NotMember {
+					Members::<T>::insert(supersig_idx, member, role);
+					number_added += 1;
+				}
+			}
+			MembersNumber::<T>::mutate(supersig_idx, |n| *n += number_added);
+
+			number_added
+		}
+
+		pub fn remove_from_supersig(
```

Comment:
rename `remove_members`

Reply:
same as add_member function, i think we need a better name. i think `remove_users` is quite good